lib/ui/classical_components/barcode_custom_ui.dart

```diff
@@ -249,11 +250,11 @@ class _BarcodeScannerWidgetState extends State<BarcodeScannerWidget> {
                 );
               }),
           showProgressBar
-              ? Center(
-                  child: Container(
+              ? const Center(
+                  child: SizedBox(
```

Comment:
This can also be const I guess

Reply:
This is done due to the automatic flutter format . and dart fix